### PR TITLE
MODINVOSTO-28/MODINVOSTO-31 Add acqUnitIds to Invoice and Voucher schemes

### PR DESCRIPTION
**Pull request by pautau-s**
## Purpose
[MODINVSTO-28](https://issues.folio.org/browse/MODINVOSTO-28)
[MODINVSTO-31](https://issues.folio.org/browse/MODINVOSTO-31)
According to [option 3](https://wiki.folio.org/display/FOLIJET/Acquisition+Units#AcquisitionUnits-OrderofOperations/Chicken&EggProblem), acquisitions-unit-assignments should become part of the invoice and voucher schemes.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Examples exist for all schemas
  - [ ] Descriptions exist for all schema properties
  - [ ] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: mod-invoice-storage/examples/voucher.sample [hunk 1]
```diff
@@ -15,6 +15,10 @@
   "type": "Payment",
   "voucherDate": "2019-05-06T00:00:00.000+0000",
   "voucherNumber": "1000",
+  "acqUnitIds": [
```
Comment (javalkar-varun) [inline mod-invoice-storage/examples/voucher.sample]: In schema `"id": "units"` so here in the sample, it should be `"units": [` ?

File: mod-invoice-storage/examples/voucher_collection.sample [hunk 2]
```diff
@@ -17,6 +17,10 @@
       "type": "Pre-payment",
       "voucherDate": "2019-05-06T00:00:00.000+0000",
       "voucherNumber": "1000",
+      "acqUnitIds": [
```
Comment (javalkar-varun) [inline mod-invoice-storage/examples/voucher_collection.sample]: In schema `"id": "units"` so here in the sample, it should be `"units": [` ?

File: mod-invoice-storage/schemas/invoice.json [hunk 3]
```diff
@@ -154,6 +154,14 @@
       "description": "This would keep the invoice from being feed into the batch process (Not generate a external voucher/payment) but would still move values in the system. Note: this is ideally defined by the vendor relationship and exposed for override on the invoice.",
       "type": "boolean"
     },
+    "acqUnitIds": {
+      "description": "acquisition unit ids associated with this invoice",
+      "id": "units",
```
Comment (javalkar-varun) [inline mod-invoice-storage/schemas/invoice.json]: You could change id to `"acqUnitIds"`, so no need to change sample files

File: mod-invoice-storage/examples/voucher.sample (hunk 1 above)
Comment (pautau-s) [reply mod-invoice-storage/examples/voucher.sample]: It must match the name of the field, not the id.

Comment (javalkar-varun) [reply mod-invoice-storage/examples/voucher.sample]: Then how will `"id"` be used?

File: mod-invoice-storage/schemas/invoice.json (hunk 3 above)
Comment (pautau-s) [reply mod-invoice-storage/schemas/invoice.json]: Removed it to not confuse anyone.

File: mod-invoice-storage/examples/voucher.sample (hunk 1 above)
Comment (pautau-s) [reply mod-invoice-storage/examples/voucher.sample]: We don't use that. Just copy-pasted from orders 🙂

Comment (javalkar-varun) [reply mod-invoice-storage/examples/voucher.sample]: 🙂

**Pull request merged**